package.json

```diff
@@ -22,7 +22,8 @@
     "request-promise": "^4.1.1",
     "ts-node": "^3.1.0",
     "tslint": "^5.4.3",
-    "typescript": "^2.4.1"
+    "typescript": "^2.4.1",
+    "strong-docs": "git://github.com/strongloop/strong-docs.git#master"
```

Comment:
@superkhau @mpatsute  I have already published strong-docs 1.3.0 and 1.3.1 with preliminary tsdoc support, see https://github.com/strongloop/strong-docs/commits/master.

I am proposing to use `^1.3.1` here, so that we can land this patch ASAP.

The fixes in strong-docs can be made independently of this pull request.

@mpatsute you can use `npm link` to install your local dev copy of strong-docs into loopback-next, so that you can verify the changes you are making in strong-docs. See https://docs.npmjs.com/cli/link